### ISSUE TITLE
zodiacal starlore

### ISSUE THREAD

**Issue by axd1967** [Stellarium/stellarium#1627]
**Is your feature request related to a problem? Please describe.**

I often want to know what the sign is of the selected date.

**Describe the solution you'd like**
A starlore that shows the abstract signs in a band along the ecliptic.

The band would 
- start from the vernal equinox 
-  extend 23.43 (declination of tropics) north and south of the ecliptic.
- It would be divided in 12 squares of 360/12 wide by 2 x 23.43 high 
- and would contain white (or some other colour) symbols in a transparent box.

![image](https://user-images.githubusercontent.com/3529789/116530915-dda95400-a8de-11eb-87f8-028df319bc7f.png)

It would also help in better recognising the astronomical symbols of the zodiacal constellations.

I think that this could be a good starter issue for someone who wants to create starlores.

(As a side issue, there exist official astronomical unicode symbols for events such as conjuntion ☌ , opposition ☍, lunar nodes ☊ ☋, planets ☉ ☿♁♂♃♄♅♆ : some of these could be very useful in the Astro calc module; tooltips could then show the normal textual meaning, and pixel space could be won)

**Describe alternatives you've considered**
A separate calendar, such as http://robinfo.oma.be/nl/efemeriden/jaar-2021/

**Additional context**

https://en.wikipedia.org/wiki/Astronomical_symbols
https://en.wikipedia.org/wiki/Astrological_symbols#Signs_of_the_zodiac

_And  no: I don't believe in astrology - just to make things clear._

**Comment by axd1967** [comment]
about the width: the proposed width is just a way to provide nearly-square cells.

an option could be provided to select the width from

- zodiac of the Moon (± 5 deg)
- zodiac of the comets
- zodiac of the planets (±8 deg)

(https://en.wikipedia.org/wiki/Zodiac#Usage)

**Comment by axd1967** [comment]
@alex-w @gzotti can we please stop this game? I'm only suggesting ideas, knowing that maybe these will help someone that is willing to implement them. it's not because I suggest ideas, that I'm obliged to implement them, wouldn't you agree?

**Comment by alex-w** [comment]
No, I disagree! If you proposed a strange or minuscule ideas, which is critical for you then you should implement it.

**Comment by gzotti** [comment]
A zodiacal SC of this appearance is quite far down the list for the core team. For my own ideas around this, I don't need the issue opened. I have not seen much activity around other people solving the issues you report either. If you need it before 2024 (or so), DIY is the fastest solution.

**Comment by Atque** [comment]
> An assignment does not stop others from doing it.

I think most people would think "aha, the feature is about to be implemented, I better wait" if they see someone assigned to the feature request/bug report.

**Comment by alex-w** [comment]
This is a good task for the community to participate in the contribution into Stellarium. Who wants to help us?

**Comment by axd1967** [comment]
Where in Transifex can I find the translations for these ?

![Image](https://github.com/user-attachments/assets/5f917559-34b0-4883-8e8f-c46d1bbb661d)
